cmd/juju/application/deployer/bundlehandler_test.go

```diff
@@ -1893,9 +1894,13 @@ relations:
     - ["wordpress:db", "mysql:server"]
 `
 	charmsPath := c.MkDir()
-	mysqlPath := testcharms.RepoWithSeries("bionic").ClonedDirPath(charmsPath, "mysql")
-	wordpressPath := testcharms.RepoWithSeries("bionic").ClonedDirPath(charmsPath, "wordpress")
-	bundle := fmt.Sprintf(content, wordpressPath, mysqlPath)
+	mysql := testcharms.RepoWithSeries("bionic").ClonedDir(charmsPath, "mysql")
```

Comment:
Let's not use Bionic, it hasn't been supported since 3.0.0. Same for all new or changed tests.

Reply:
It's disappointing. Is there a different charm you can use.

Reply:
There are only 3 charms in the focal dir, one of them might be a good subsituite.

We use the `mysql` charm in the `testcharms/charm-repo/bionic` and it's duplicate in `/testcharms/charm-repo/quantal` in a lot of places. It could always be duplicated again into focal (or a new `jammy` dir) if not

This whole system probably needs updating, why are our test charms organised into subdirs with these arbitrary strings?